### PR TITLE
Feat : Added Text utilities

### PR DESCRIPTION
**Pull request by srejitk**
Added Text Utilities applicable for the Component Library
Typo

### CODE REVIEW

File: stylesheets/typography.css [hunk 1]
```diff
@@ -1,72 +1,113 @@
...
 .material-icons {
     font-family: 'Material Icons';
     font-weight: normal;
     font-style: normal;
-    font-size: 24px;  /* Preferred icon size */
+    font-size: 24px;
+    /* Preferred icon size */
     display: inline-block;
     line-height: 1;
     text-transform: none;
```
Comment (rahulcs754) [inline stylesheets/typography.css]: if you are not using text-transform other value than why are you use text-transform: none 

none is default value of text-transform

File: stylesheets/typography.css [hunk 2]
```diff
@@ -1,72 +1,113 @@
...
-    letter-spacing: normal;
     word-wrap: normal;
     white-space: nowrap;
     direction: ltr;
```
Comment (rahulcs754) [inline stylesheets/typography.css]: here direaction has default value is ltr why are you set default value?

**Review by rahulcs754 — COMMENTED** [review]
I looked at your code and you used mostly default values which are not good. if any CSS property has a default value then don't use it.

File: stylesheets/typography.css (hunk 1 above)
Comment (kjais1720) [inline stylesheets/typography.css]: If you are not self hosting the material icons, then this class is not needed, it is automatically imported from the CDN link.

**Review by kjais1720 — APPROVED** [review]
All good, just a suggestion, you can remove the `.material-icons` class if you are using CDN to import the icons

File: stylesheets/typography.css (hunk 2 above)
Comment (srejitk) [reply stylesheets/typography.css]: This was part of the material Icons code. Have removed it and added CDN Dependency instead.

File: stylesheets/typography.css (hunk 1 above)
Comment (srejitk) [reply stylesheets/typography.css]: Didn't know this. Thank Krituraj. Have corrected in the latest commit.

Comment (srejitk) [reply stylesheets/typography.css]: Corrected in the latest commit Rahul. Thanks for bring this to notice.

**Pull request merged**